apps/web/.env.example

```diff
@@ -49,4 +49,6 @@ PROJECT_ID_VERCEL=
 TEAM_ID_VERCEL=
 VERCEL_AUTH_BEARER_TOKEN=
 
-NEXT_PUBLIC_URL=
\ No newline at end of file
+NEXT_PUBLIC_URL=
+
+NEXT_PUBLIC_SENTRY_DSN=''
```

Comment:
Let's add `SENTRY_AUTH_TOKEN` as well (including `env.ts`)

Reply:
that's only to publish the source map to sentry not sure if we really need it